src/components/Tenant/TenantCriteria/style.jsx

```diff
@@ -0,0 +1,132 @@
+import styled from 'styled-components';
+
+const STenantCriteria = styled.div`
+  background-color: #ffffff;
+  width: 1100px;
+  margin: 0 auto;
+   {
+    /* Text group*/
+  }
+  .titleCriteria {
+    font-family: Roboto;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 60px;
+    line-height: 70px;
+    margin-left: 6vw;
+    margin-top: 5vh;
+  }
+  .titleTown {
+    font-family: Roboto;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 20px;
+    line-height: 29px;
+  }
+  .titlePets {
+    font-family: Roboto;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 20px;
+    line-height: 29px;
+  }
+  .titleCoTenant {
+    font-family: Roboto;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 20px;
+    line-height: 29px;
+  }
+  .titleNumberCoTenant {
+    font-family: Roboto;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 20px;
+  }
+  .titleYarsCoTenant {
+    font-family: Roboto;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 20px;
+  }
+  .titleAgeCoTenant {
+    font-family: Roboto;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 20px;
+  }
```

Comment:
Here, create a global class 'title' for : font-family, font-style, font-weight and font-size is better I think. (I dev I faignasse)
`className="title pets"` etc..